### ISSUE TITLE
pose training mAP？

### ISSUE THREAD

**Issue by wzf19947** [ultralytics/ultralytics#2160]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

Ｉ‘ｍ　ｔｒａｉｎｉｎｇ　ａ　ｐｏｓｅ　ｍｏｄｅｌ　ｗｉｔｈ　ｍｙ　ｃｕｓｔｏｍ　ｙｍａｌ　ｆｒｏｍ　scratch　ｕｓｉｎｇ　ＣＯＣＯ　ｄａｔａｓｅｔｓ，ｗｈｅｎ　Ｉ　ｔｒａｉｎｅｄ　ａｂｏｕｔ　２０　ｅｐｏｃｈｓ，ｔｈｅ　ｐｏｓｅ　ｍＡＰ　ｉｓ　ｖｅｒｙ　ｌｏｗ　ｗｈｉｌｅ　ｔｈｅ　ｄｅｔｅｃｔ　ｍＡＰ　ｉｓ　ａｂｏｕｔ　０．８６，　ｈｏｗ＇ｓ　ｉｔ　ｈａｐｐｅｎ？　Ｉ　ｔｒａｉｎｅｄ　ｗｉｔｈ　ｃｍｄ：　ｙｏｌｏ　ｃｆｇ＝ｄｅｆａｕｌｔ＿ｃｆｇ
![IMG_20230421_092931](https://user-images.githubusercontent.com/30069226/233527237-79ad5e31-bc8f-4734-88df-299ff8be23f0.jpg)

**Comment by Laughing-q** [comment]
@wzf19947 this could happen since detection usually converge faster than pose. And I strongly recommend you to use a pretrained model as an initialized weight for faster  convergence.

**Comment by wzf19947** [comment]
> @wzf19947 this could happen since detection usually converge faster than pose. And I strongly recommend you to use a pretrained model as an initialized weight for faster convergence.

I kept training with about 120 epochs, and I found the mAP of Pose almost never  change, about 0.0004.  I didn't change any code, and I'm really confusing if I trained it the wright way, cause it takes about have a week training with my poor GPU.

**Comment by Laughing-q** [comment]
@wzf19947 are you training with official model or your own modified model yaml?

**Comment by Laughing-q** [comment]
@wzf19947 could you also share your config file?

**Comment by wzf19947** [comment]
> @wzf19947 could you also share your config file?

I used with my own modified model yaml , something like yolov5n.yaml and changed the Detect to Pose head ,nothing special. Detect branch works fine when predict ,but couldn't predict any keypoint at all.  Maybe I'll train a official model  to see if my project is working right. @Laughing-q

**Comment by Laughing-q** [comment]
@wzf19947 yeah that would be a way, and also please check your `train_batch*.jpg` to make sure all labels(both bbox and kpts) are good.
